verilog/tools/ls/verilog_ls.cc

```diff
@@ -146,7 +151,24 @@ int main(int argc, char *argv[]) {
   // -- Register JSON RPC callbacks
 
   // Exchange of capabilities.
-  dispatcher.AddRequestHandler("initialize", InitializeServer);
+  dispatcher.AddRequestHandler(
+      "initialize",
+      [&symboltablehandler](const verible::lsp::InitializeParams &p) {
+        // TODO add filling the include_paths and corpus
+        std::cerr << "Initialize workspace paths:  rootPath='" << p.rootPath
+                  << "' rootUri='" << p.rootUri << "'" << std::endl;
+        if (!p.rootUri.empty()) {
+          symboltablehandler.setProject(p.rootUri, {}, "");
```

Comment:
You probably also want it to be possible for the user to initialize a file-list beforehand via command-line flags; look for `AppendFileListFromCommandline()`.

Then, additional information can be added in this call.

Reply:
I will add reading the include paths from command-line flags